packages/@okta/vuepress-theme-prose/assets/css/abstracts/_colors.scss

```diff
@@ -56,15 +83,15 @@ $colors: (
     'whale': #142430,
   ),
   'gray': (
-    '000': #fafafa,
-    '100': #e4e5e7,
-    '200': #cdd1d3,
-    '300': #b7bcc0,
-    '400': #a0a7ac,
-    '500': #899298,
-    '600': #737d85,
-    '700': #5c6971,
-    '800': #46545e,
-    '900': #2f3f4a,
+    '000': #ffffff,
+    '100': #EBEBED,
+    '200': #DCDCDC,
+    '300': #DCDCDC,
+    '400': #C1C1C8,
+    '500': #6E6E78,
+    '600': #6E6E78,
+    '700': #6E6E78,
+    '800': #1D1D21,
+    '900': #1D1D21,
```

Comment:
Circling back on @alexdahl-okta's previous comment, should these values be the same as odyssey's?

For design, was this redesign done with Odyssey's branding in mind? @alexdahl-okta

Reply:
Yes, design intent is to use real actual Odyssey. I don't think this re-skin should have any custom colors.

(Odyssey color [docs](https://odyssey.okta.design/base/color/), [source](https://github.com/okta/odyssey/blob/develop/packages/odyssey/src/scss/abstracts/_colors.scss).)

I'll defer to engineering, but I would find it confusing for tokens here to differ from tokens in Odyssey. For example, Gray 000 is #f5f5f6 – white is a separate token.

(imho, using the actual Odyssey npm package would better than trying copy and paste values.)

Reply:
@ulyanash it would be best if we can use the values and macro from the Odyssey package that Alex linked. Can you take a look and see if we can use those wherever possible?